hubs.yaml

```
@@ -7,6 +7,9 @@ clusters:
       project: two-eye-two-see
       cluster: low-touch-hubs-cluster
       zone: us-central1
+      nfs:
+        server: nfs-server-01
+        zone: us-central1-b
```

I'm not sure about this, but shouldn't the dir be created in the zone where the project is running? 

```suggestion
        zone: us-central1
```

---

So GKE clusters can be [regional or zonal](https://cloud.google.com/kubernetes-engine/docs/concepts/types-of-clusters). Regional clusters have masters running in 3 zones in the same region for resiliency, but cost more. In some cases, it's worth it and in others it isn't. I've now updated the code with a commit that specifies that more explicitly for the cluster.

Our NFS server is just a single machine though, and they only exist in one zone. Even though our cluster is regional, all our user nodes are only in one zone (us-central1-b), so our NFS machine is there too.

You can see some of the config that creates all this [in our terraform](https://github.com/2i2c-org/pilot-hubs/blob/master/terraform/main.tf#L43)